silkrpc/commands/ots_api.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+    Copyright 2020 The Silkrpc Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#include "ots_api.hpp"
+
+#include <silkrpc/json/types.hpp>
+
+#include "silkrpc/core/blocks.hpp"
+#include "silkrpc/core/state_reader.hpp"
+#include "silkrpc/ethdb/kv/cached_database.hpp"
+#include "silkrpc/ethdb/transaction_database.hpp"
+
+namespace silkrpc::commands {
+
+boost::asio::awaitable<void> OtsRpcApi::handle_ots_get_api_level(const nlohmann::json& request, nlohmann::json& reply) {
+    reply = make_json_content(request["id"], 8);
```

Comment:
Please add a constant here instead of hard-coded value, something like:
```
constexpr int kCurrentApiLevel{8};
```
You can put the constant inside the namespace before the `handle_ots_get_api_level` method definition.

Reply:
Ok